libraries/AP_RangeFinder/AP_RangeFinder_PulsedLightLRF.cpp

```diff
@@ -140,6 +146,17 @@ static const struct settings_table settings_v2[] = {
     { LL40LS_MEASURE_REG, LL40LS_MSRREG_ACQUIRE },
 };
 
+/*
+  register setup table for V3HP Lidar
+ */
+static const struct settings_table settings_v3hp[] = {
+    { LL40LS_MEASURE_REG, LL40LS_MSRREG_ACQUIRE },
+    { LL40LS_SIG_COUNT_MAX, 0x80},
+    { LL40LS_ACQ_CONFIG_REG, LL40LS_EN_QUICKTERM},
```

Comment:
What is the reason for enabling quick termination? This was already available with v3 but not used. As we don't have continuous measurement now, it seems less likely **to me** that we need quick termination (it is less accurate at short distances in this mode).

Reply:
<img width="511" alt="screen shot 2018-09-08 at 3 01 25 pm" src="https://user-images.githubusercontent.com/15794355/45251884-13891b00-b378-11e8-80e4-133ae42c181c.png">
At first, I thought we need quick termination because we are sampling at high speed (50Hz) so the measurement duration must be as fast as possible, but after re-calculating the pulse acquisition time, I have removed quick termination. Specifically, we choose 128 acquisitions (LL40LS_SIG_COUNT_MAX, 0x80), at repetition frequency of 10-20kHz (datasheet). Therefore, the measurement duration should be around 6.4ms to 12.8ms (78-156Hz). It is fast enough so we don't need quick termination.